pandas/core/frame.py

```diff
@@ -1697,19 +1697,36 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
 
         .. versionadded:: 0.21.0
 
+        Requires either fastparquet or pyarrow libraries.
+
         Parameters
         ----------
         fname : str
-            string file path
+            String file path.
         engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
             Parquet library to use. If 'auto', then the option
             ``io.parquet.engine`` is used. The default ``io.parquet.engine``
             behavior is to try 'pyarrow', falling back to 'fastparquet' if
             'pyarrow' is unavailable.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
-        kwargs
-            Additional keyword arguments passed to the engine
+        kwargs : dict
+            Additional keyword arguments passed to the engine.
+
+        Examples
+        ----------
+        >>> df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.to_parquet('df.parquet.gzip', compression='gzip')
+
+        Returns
+        ----------
+        Nothing.
+
+        See Also
+        --------
+        DataFrame.to_csv : write a csv file.
+        DataFrame.to_sql : write to a sql table.
+        DataFrame.to_hdf : write to hdf.
```

Comment:
For the "See Also" section, please check the section 5 of https://python-sprints.github.io/pandas/guide/pandas_docstring.html